### PR TITLE
Added exposion proof items

### PR DESCRIPTION
**Pull request by Hermelijn15**
Ignore explosions for a specified list of Item entity's
should now be in the correct patch files

### CODE REVIEW

File: patches/server/0033-Added-list-of-ItemEntity-s-that-ignore-explosions.patch [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+diff --git a/src/main/java/net/minecraft/world/entity/Entity.java b/src/main/java/net/minecraft/world/entity/Entity.java
+index 50259042ebafe520342bdb1a4b7e6b9138b8acbd..ce082ea8e401ac37c32f6418ecc41a51e235e1aa 100644
+--- a/src/main/java/net/minecraft/world/entity/Entity.java
++++ b/src/main/java/net/minecraft/world/entity/Entity.java
+@@ -4411,6 +4411,12 @@ public abstract class Entity implements Nameable, EntityAccess, CommandSource {
+         return InteractionResult.PASS;
+     }
+ 
++    // Sakura start - getter to get level in ItemEntity.java
```
Comment (Samsuik) [inline patches/server/0033-Added-list-of-ItemEntity-s-that-ignore-explosions.patch]: This isn't necessary. You can get the entity level by calling `level()` within ItemEntity.

**Review by Samsuik — COMMENTED** [review]
The configuration looks good.

File: patches/server/0033-Added-list-of-ItemEntity-s-that-ignore-explosions.patch (hunk 1 above)
Comment (Hermelijn15) [reply patches/server/0033-Added-list-of-ItemEntity-s-that-ignore-explosions.patch]: even when it is private? I will try tomorrow

Comment (Samsuik) [reply patches/server/0033-Added-list-of-ItemEntity-s-that-ignore-explosions.patch]: Only the field is private the `level()` accessor is public

Comment (Hermelijn15) [reply patches/server/0033-Added-list-of-ItemEntity-s-that-ignore-explosions.patch]: ah makes sense

**Pull request merged**